Mathlib/FieldTheory/Finite/Basic.lean

```diff
@@ -201,6 +228,84 @@ theorem sum_pow_units [DecidableEq K] (i : ℕ) :
         apply forall_congr'; intro x; simp [Units.ext_iff]
 #align finite_field.sum_pow_units FiniteField.sum_pow_units
 
+/-- The sum of a nontrivial subgroup of the units of a field is zero. -/
+theorem sum_subgroup_units_zero_of_ne_bot {G : Subgroup Kˣ} [Fintype G] (hg : G ≠ ⊥) :
```

Comment:
```suggestion
theorem sum_subgroup_units_eq_zero {G : Subgroup Kˣ} [Fintype G] (hg : G ≠ ⊥) :
```

I'm sure that the `of_ne_bot` can be figured out later, and usually we spell these theorems with `_eq`.